### ISSUE TITLE
[T|Server] Wait until tarantool started in run_tests.sh

### ISSUE THREAD

**Issue by alexeyknyshev** [alexeyknyshev/android_cash_points#16]
At the moment in **./run_tests.sh** `sleep 5` gives timeout for tarantool to start. It's **ugly** hack.
Please, reimplement this logic in **./run_tests.sh** in any of this ways:

- Periodically (let every 1 sec) read contents of  **tnt_workdir/tnt.log** and don't start tests until such line:
`2016-03-10 11:57:24.852 [23127] main/101/init.lua I> ready to accept requests`
- Periodically (let every 1 sec) try to connect to tarantool via **nc** (admin console) until success
- Figure out better way to do this task (**preferred**). Maybe something from [this doc](http://tarantool.org/doc/book/administration.html)

**Comment by alexeyknyshev** [comment]
https://github.com/alexeyknyshev/android_cash_points/blob/master/server/run_tests.sh#L111

**Comment by alexeyknyshev** [comment]
In addition. 
Add into **run_tests.sh** logic that would allow use **already started** tarantool instance on localhost.
To do this:
- check existing instance via `pidof tarantool `
- if instance exists don't start another one, otherwise start new tarantool instance (already done in script) and kill it on exit

**Comment by alexeyknyshev** [comment]
Done by @CloudedCat in pr #22
